routes/weather.js

```diff
@@ -147,7 +147,10 @@ router.get('/forecast/:zipcode', (request, res) => {
                     })
                     let weatherDescription = forecast[5].weather[0].description // Get the weather description for 12:00pm-3:00pm interval
                     let icon = forecast[5].weather[0].icon
-                    let weatherDate = (forecast[5].dt_txt).slice(5,10) //Get the date as only mm-dd format
+                    //let weatherDate = (forecast[5].dt_txt).slice(5,10) //Get the date as only mm-dd format
+                    var d = new Date(forecast[5].dt_txt);
+                    var days = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+                    let weatherDate = days[d.getDay()];
```

Comment:
Won't d.getDay() return back a specific day like the 12th or the 4th? If so how can that work by indexing the days array?

Reply:
Also can you run heroku local, and then hit the /forecast/zipcode endpoint and take a screenshot and send in the chat so I can see the result

Reply:
d.getDay returns and int from 0-6 which is from sunday to monday.

Reply:
gotcha

Reply:
also next time i would avoid using the variable name 'd'. Should be more descrptive of what it is

Reply:
I got how the getDay() method works from the internet and forgot to change the variable name. I got it fixed now.